locations/spiders/carhartt_wip.py

```diff
@@ -0,0 +1,74 @@
+                    }
+                ),
+                headers={
+                    "Content-Type": "application/json",
+                },
+                callback=self.parse,
+            )
+
+    def parse(self, response):
+        data = response.json()
+        store_data = data.get("data").get("stores").get("entities")
+
+        for store in store_data:
+            oh = OpeningHours()
+            day_data = store.get("hours") or {}
+            day_dict = dict(day_data)
+            for day, value in day_dict.items():
+                if isinstance(value, str) or value.get("isClosed"):
+                    continue
+                for interval in value.get("openIntervals"):
+                    oh.add_range(
+                        day=day.title()[:2],
+                        open_time=interval.get("start"),
+                        close_time=interval.get("end"),
+                        time_format="%H:%M",
+                    )
+            emails = store.get("emails")
+            properties = {
+                "ref": store.get("meta").get("uid"),
+                "name": store.get("name"),
+                "street_address": store.get("address").get("line1"),
```

Comment:
```suggestion
                "street_address": ", ".join(
                    filter(
                        None,
                        [store["address"].get("line1"), store["address"].get("line2")],
                    )
                ),
```

This isn't a clear win as line2 isn't great, sometimes it includes opening hours. No strong opinion on if you accept this change.